mdaviz/mda_folder.py

```diff
@@ -76,6 +76,16 @@ def doFileSelected(self, index):
             self.select_fields_tableview.selected.connect(self.doPlot)
             self.setStatus(f"Selected file: {self.mdaFileList()[index.row()]}")
 
+            if self.mda_file_visualization.isPlotBlankQt():  # TODO what about MPL?
+                # TODO: this should depend on the selection: auto-replace vs auto-add
+                first_pos_idx = self.select_fields_tableview.firstPos()
+                first_det_idx = self.select_fields_tableview.firstDet()
+                if first_pos_idx is not None and first_det_idx is not None:
+                    first_selections = {"X": first_pos_idx, "Y": [first_det_idx]}
+                    self.doPlot("replace", first_selections)
```

Comment:
Does `doPlot()` handle question of Qt or Mpl?  Can it relieve the above TODO marker?

Reply:
create a new issue #55